README.md

```diff
@@ -55,12 +55,12 @@ func main() {
   ts := oauth2.StaticTokenSource(
     &oauth2.Token{AccessToken: "... your access token ..."},
   )
-  tc := oauth2.NewClient(oauth2.NoContext, ts)
+  tc := oauth2.NewClient(context.Background(), ts)
```

Comment:
What about adding a line before this:
`ctx := context.Background()`
and then changing this line to use `ctx` ?